quest/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "2.4.1" %}
+
+package:
+  name: quest
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erdc/quest.git
+  git_rev: {{ version }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pbr
+    - pytest-runner
+
+  run:
+    - python
+    - fiona
+    - rasterio
+    - gdal
+    - numpy
+    - geojson
+    - jinja2
+    - ulmo
+    - pyyaml
+    - matplotlib
+    - click
+    - werkzeug
+    - peewee
+    - stevedore
+    - geopandas
+    - distributed
+    - psutil
+    - pint
+    - pony
+    - tornado
+    - sortedcollections
+    - pytest-cov
+    - param
+    - sphinx
+    - sphinx_rtd_theme
+    - hs_restclient
+    - girder-client
+
+test:
+  imports:
+    - quest
+
+about:
+  home: https://github.com/erdc/quest
+  license: BSD
+  summary: Quest is a python library that provides an API the ability to search, 
+  publish and download data (both geographical and non-geographical) across multiple 
+  data sources including both local repositories and web based services. The library 
+  also allows provides tools in order to manipulate and manage the data that the user 
+  is working with.
```

Comment:
Please pick a [YAML text wrapping format]( https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines/21699210#21699210 ). There are many to choose from (unfortunately this is not one). That said, they are all pretty easy to do and should be minor tweaks on what you have currently.

Reply:
There is an example of this in the example, too! https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L65